src/pages/result/Result.jsx

```diff
@@ -1,13 +1,33 @@
 import React from 'react';
 import { useCategory } from '../../utils/category-context';
+import { quizData } from "../../data/quizData.js";
 import "./result.css";
+import { Link } from 'react-router-dom';
 
 function Result() {
   const { selectedCategory} = useCategory();
+  const {quiz} = quizData.find(
+    (element) => element.categoryValue === selectedCategory.value
+  );
+
   return (
-    <div className='my-4 flex flex-col items-center'>
+    <div className='my-4 flex flex-col'>
       <h2 className='result-heading my-4'>Quiz Result</h2>
-      <h4 className='my-4'>Your Score: <span className='score'>{selectedCategory.score}</span></h4>
+      <h3 className='result-score my-8 mx-auto p-2 border-5'>You Scored: <span className='score'>{selectedCategory.score}</span>/50</h3>
+      {
+        quiz.map((item) => <div key={item.question} className='ques-wrapper'>
+          <h4>Q. {item.question}</h4>
+          {item.options.map((option) => <p
+          key={option.answer}
+          className={`my-4 ${option.isCorrect ? "option-correct" : ""}`}
```

Comment:
instead of using ternary you can try with short-circuiting, as you are not set any else condition.
`${option.isCorrect && "option-correct" }`

Reply:
done changes ✅